Makefile

```diff
@@ -28,11 +28,11 @@ MANPAGES_MD = $(wildcard docs/*.md)
 
 all: binary docs
 
-binary: skopeo
+binary: skopeo-build
 
 # Build a docker image (skopeobuild) that has everything we need to build.
 # Then do the build and the output (skopeo) should appear in current dir
-skopeo: cmd/skopeo
+skopeo-build: cmd/skopeo
 	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
```

Comment:
The failure case is make the skopeo binary.. touch a source file.. make the skopeo binary. Make fails to build the skopeo binary because it does not check the skopeo binary against the source tree. Required make clean in between builds.